extensions/wikia/PortableInfobox/tests/InfoboxParamsValidatorTest.php

```diff
@@ -4,6 +4,7 @@ class InfoboxParamsValidatorTest extends WikiaBaseTest {
 	private $InfoboxParamsValidator;
 	private $invalidParamsExpectionName =
 		'Wikia\PortableInfobox\Helpers\InvalidInfoboxParamsException';
+	private $invalidColorValueExceptionName = 'Wikia\PortableInfobox\Helpers\InvalidColorValueException';
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "$invalidColorValueExceptionName" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1068)